src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -270,12 +277,12 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     }
 
     LogPrint("zrpc", "%s: spending %s to send %s with fee %s\n",
```

Comment:
Non-blocking: I think this might need to be `zrpcunsafe`, depending on the source/destination (based on my understanding that `zrpcunsafe` is for anything that logs data which would otherwise be encrypted on the block chain).